sqs/src/main/scala/akka/stream/alpakka/sqs/SqsBatchException.scala

```diff
@@ -6,8 +6,8 @@ package akka.stream.alpakka.sqs
 
 import akka.annotation.InternalApi
 
-final class SqsBatchException @InternalApi private[sqs] (val batchSize: Int, message: String)
-    extends Exception(message) {
+final class SqsBatchException @InternalApi private[sqs] (val batchSize: Int, message: String, cause: Throwable = null)
```

Comment:
Instead of having `null` as a default argument it would be better to have either multiple constructors that take or not the `cause`, or have one constructor that takes an `Option`.

Reply:
fixed